Description: Removes lockfile from requirements.txt
 This dependency is in fact wrong, and murano doesn't need it. Since this is a
 problematic lib which is too unstable (last version breaks API), it is best
 to actually remove it.
Author: Thomas Goirand <zigo@debian.org>
Bug-Debian: https://bugs.debian.org/789888
Forwarded: https://review.openstack.org/#/c/194247/
Last-Update: 2015-06-26

--- murano-2015.1.0.orig/requirements.txt
+++ murano-2015.1.0/requirements.txt
@@ -11,7 +11,6 @@ PasteDeploy>=1.5.0
 Routes>=1.12.3,!=2.0
 WebOb>=1.2.3
 kombu>=2.5.0
-lockfile>=0.8
 psutil>=1.1.1,<2.0.0
 iso8601>=0.1.9
 six>=1.9.0
